app/scripts/migrations/107.ts

```diff
@@ -0,0 +1,82 @@
+import { cloneDeep } from 'lodash';
+import { hasProperty, isObject } from '@metamask/utils';
+
+export const version = 107;
+
+interface Account {
+  address: string;
```

Comment:
don't think this is correct https://github.com/MetaMask/core/blob/76787ceeab5083780f50a2f9e68617f08cc600c4/packages/assets-controllers/src/AccountTrackerController.ts#L29

Reply:
I'm big dumb